matlab/src/Connection.cpp

```diff
@@ -17,8 +17,6 @@ namespace
     {
         Type = 0,
         Underlying,
-        Incoming,
```

Comment:
We only support outgoing connections with MATLAB. And we don't support "bidir" with MATLAB.